docs/changelogs/v0.20.md

```diff
@@ -6,13 +6,110 @@
 
 - [Overview](#overview)
 - [🔦 Highlights](#-highlights)
+  - [Boxo under the covers](#boxo-under-the-covers)
+  - [HTTP Gateway](#http-gateway)
+    - [Switch to `boxo/gateway` library](#switch-to-boxogateway-library)
+    - [Improved testing](#improved-testing)
+    - [Trace Context support](#trace-context-support)
+    - [Removed legacy features](#removed-legacy-features)
```

Comment:
@BigLep I've added some notes about most important changes
@hacdias feel free to add if there was anything important I've missed (

Reply:
I see there's no mention regarding `--empty-repo` becoming the default. It's not a "removed feature", but I'm not sure whether it should get into this list of highlights or not.

Reply:
Per 2023-04-13 maintainer conversation, @hacdias is going to add this.